app/controllers/registrations_controller.rb

```diff
@@ -104,4 +104,8 @@ def update_password
     @user = User.find_by(id: params[:id])
     redirect_to login_path, alert: t('errors.users.not_found') unless @user
   end
+
+  private def set_cookies
+    cookies[:auth_token] = params[:remember_me] == '1' ? { value: @user.auth_token, expires: 10.days.from_now } : @user.auth_token
```

Comment:
when will it get expire when we don't set expires ?

Reply:
Sir it will expire when session is over (i.e. when we close the browser)